tmt/steps/provision/testcloud.py

```diff
@@ -655,6 +666,16 @@ def guest(self) -> Optional[tmt.Guest]:
         """ Return the provisioned guest """
         return self._guest
 
+    def _print_local_images(self) -> None:
+        """ Print images which are already cached """
+        self.info("Locally available images including file:// prefix")
+        try:
+            for f in os.listdir(TESTCLOUD_IMAGES):
```

Comment:
Since `TESTCLOUD_IMAGES` is `Path`, we can use the native `TESTCLOUD_IMAGES.iterdir()`.

Also, I’d pickndifferent name instead of `f` - it’s often used for open files, how about `filename`?

Reply:
good points.

Reply:
What do you think about `TESTCLOUD_IMAGE.glob()`? Using this, we can also skip the if statement on the line below. Pls see: https://github.com/teemtee/tmt/pull/1997/commits/38600f12c7ce9a4decf6cc826b14d98efb0ec719

Reply:
Nice